buku.1

```diff
@@ -214,6 +214,18 @@ Show selective monochrome output with specific fields. Works with --print. Searc
 .br
 .I N
 = 4, show URL, title and tags in a single line
+.br
```

Comment:
Combine the rest 4 similar to the help text. No need to add each of 10, 20, 30, 40...